presidio-analyzer/presidio_analyzer/analyzer_loader.py

```diff
@@ -0,0 +1,72 @@
+import yaml
+import os
+import logging
+from typing import List
+
+from presidio_analyzer import AnalyzerEngine, RecognizerRegistry
+from presidio_analyzer.nlp_engine import NlpEngineProvider
+import presidio_analyzer.recognizer_registry
```

Comment:
How is this import used? The two objects (AnalyzerEngine, RecognizerRegistry) are already loaded in line 6.

Reply:
I used to need it when dynamically instantiating the recognizers, might not need it now, let me check :D

Reply:
Yeah no I still need it